example/src/main/java/com/midtrans/sample/controller/MobileSdkController.java

```diff
@@ -1,30 +1,84 @@
 package com.midtrans.sample.controller;
 
 import com.midtrans.Config;
-import com.midtrans.ConfigFactory;
 import com.midtrans.httpclient.error.MidtransError;
-import com.midtrans.service.MidtransSnapApi;
+import com.midtrans.httpclient.SnapApi;
+import com.midtrans.httpclient.TransactionApi;
 import org.json.JSONObject;
+import org.springframework.http.HttpStatus;
 import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.PostMapping;
 import org.springframework.web.bind.annotation.RequestBody;
 import org.springframework.web.bind.annotation.RestController;
 
 import java.util.Map;
 
+import static com.midtrans.sample.data.Constant.sandboxServerKey;
+
 @RestController
 public class MobileSdkController {
 
-    private MidtransSnapApi snapApi = new ConfigFactory(
-            new Config("SB-Mid-server-TOq1a2AVuiyhhOjvfs3U_KeO",
-                    "SB-Mid-client-nKsqvar5cn60u2Lv",
-                    false))
-            .getSnapApi();
+    /**
+     * Midtrans java sample use, Using static method and Midtrans Config {@link Config}.
+     * This is a new static method, the config will use for request Snap token on Midtrans API static (SnapApi class).
+     * {@link SnapApi}
+     * Sample use on mobile sdk Controller @line 52
+     *
+     * also you can put others config, like enableLog, timeConnect, proxy, idempotent-key, etc.
+     */
+    private Config configOptions = Config.builder()
+            .setSERVER_KEY(sandboxServerKey)
+            .setIsProduction(false)
+            .setPaymentOverrideNotification("https://midtrans-java.herokuapp.com/mobile-notification")
+            .build();
+
 
     // API `/charge` for mobile SDK to get Snap Token
     @PostMapping(value = "/charge", produces = MediaType.APPLICATION_JSON_VALUE)
-    public Object createTokeSnap(@RequestBody Map<String, Object> body) throws MidtransError {
-        JSONObject result = snapApi.createTransaction(body);
-        return result.toString();
+    public ResponseEntity<Object> createTokeSnap(@RequestBody Map<String, Object> body) {
+        try {
+            JSONObject result = SnapApi.createTransaction(body, configOptions);
+            return new ResponseEntity<>(result.toString(), HttpStatus.OK);
+        } catch (MidtransError midtransError) {
+            midtransError.printStackTrace();
+            return new ResponseEntity<>(HttpStatus.INTERNAL_SERVER_ERROR);
+        }
+    }
+
+    @PostMapping(value = "/mobile-notification", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
for backlog:
- consider renaming path to `/payment-status-notification-handler`